src/app/components/campus/campus.component.ts

```diff
@@ -0,0 +1,53 @@
+
+
+import { Component, OnInit } from '@angular/core';
+
+import { SubscriptionsComponent } from '../../common/subscriptions-component';
+
+import { IImage} from './modules/slideshow/IImage';
```

Comment:
You should be able to `import { IImage } from 'ng-simple-slideshow';` and safely remove the other file.